prijateli_tree/app/database.py

```diff
@@ -180,12 +180,10 @@ class Player(Base):
     name_hidden = Column(Boolean, nullable=False)
     game = relationship(
         "Game",
-        foreign_keys="[Player.user_id, Player.game_id]",
         back_populates="players",
     )
     answers = relationship(
         "GameAnswer",
-        foreign_keys="[Player.user_id, Player.game_id]",
```

Comment:
These changes should be addressed in this PR: https://github.com/prijatelilab/PrijateliTree/pull/24

I don't want to remove the FK constraints, I'd rather get to the bottom of why things are not working.